app/views/tasks/_in_progress_tasks.html.erb

```diff
@@ -6,7 +6,12 @@
     <div class="panel-body">
       <% tasks.each do |task| %>
           <ul class="list-group">
-            <li class="list-group-item"><%= task.name %>
+            <li class="list-group-item"><%= task.name %><%= task.started_at %>
```

Comment:
as for me we don't need to display started at here, because we don't have enough space.
But as you wish :-)